tools/data/kinetics/extract_frames.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+DATASET=$1
+
+cd ../
+python build_rawframes.py ../../data/${DATASET}/videos_train/ ../../data/${DATASET}/rawframes_train/ --level 2 --flow-type tvl1 --ext mp4 --task both  --new-width 340 --new-height 256
```

Comment:
add a note in preparing_kinetics.md, to remind users that it is using the fixed resolution of 340x256.
users can change the script if they want other options.

Reply:
There already is a note in preparing_kinetics.md, at the end of Step 3.